src/legacy/ui/public/fancy_forms/kbn_form_controller.js

```diff
@@ -53,11 +53,9 @@ export function decorateFormController($delegate, $injector) {
     }
 
     describeErrors() {
-      const count = this.softErrorCount();
       return i18n.translate('common.ui.fancyForm.errorDescription',
         {
-          defaultMessage: '{count, plural, one {# Error} other {# Errors}}',
-          values: { count }
+          defaultMessage: 'Errors'
```

Comment:
Let's try to be a bit more verbose here:

`Errors in the highlighted fields need to be resolved.`

Reply:
Updated.